scripts/record-build-success.ts

```diff
@@ -23,35 +23,22 @@ All portions of the code written by the Ethereal Engine team are Copyright © 20
 Ethereal Engine. All Rights Reserved.
 */
 
-import { Static, Type } from '@feathersjs/typebox'
 import appRootPath from 'app-root-path'
 import cli from 'cli'
 import dotenv from 'dotenv-flow'
 import fs from 'fs'
 import knex from 'knex'
 
+import {
```

Comment:
Have you tried running this file with this import? I hope it doesn't give esm import errors.

Just wandering if we have to import it like done in `getClientSettings.ts` file like:
```
const {
  clientSettingPath,
  ClientSettingDatabaseType
} = require('../../engine/src/schemas/setting/client-setting.schema')
```

Reply:
Yes, I tested it on dev and it was fine. We use relative paths if the file to be imported is in the same package as the file importing it, and absolute paths if it's a top-level script or importing from a different package.